cypress/obj/TimeLoggingPage.js

```diff
@@ -0,0 +1,27 @@
+class TimeLoggingPage {
+    getUrl(){
+        return cy.url()
```

Comment:
This one is not really a locator or has anything specific to timeLoggingPage. It is a standard cypress function and could "live" in a test case